view/adminhtml/templates/selectwidget.phtml

```diff
@@ -131,214 +131,197 @@ input:checked + .slider:before {
+                if(this.checked){
+                    jQuery("#exlucded_urls_container").show();
+                    jQuery('#donotdisplay').prop('checked', false);
+                    jQuery("#included_urls_container").hide();
+                }else{
+                    jQuery("#exlucded_urls_container").hide();
+                    jQuery('#donotdisplay').prop('checked', true);
+                    jQuery("#included_urls_container").show();
+                }
+            });
+
+            jQuery("#donotdisplay").change(function() {
+                if(this.checked){
+                    jQuery("#included_urls_container").show();
+                    jQuery('#alwaysdisplay').prop('checked', false);
+                    jQuery("#exlucded_urls_container").hide();
+                }else{
+                    jQuery("#included_urls_container").hide();
+                    jQuery('#alwaysdisplay').prop('checked', true);
+                    jQuery("#exlucded_urls_container").show();
+                }
+            });
+        });
+
+        function displayWidget (websiteId) {
+            var storedWidgetUrl = '<?= $block->escapeUrl($block->getStoreWidget()) ?>';
+            var domain = '<?= $block->escapeUrl($block->mainurl()) ?>';
+            jQuery.get(storedWidgetUrl + '?id=' + websiteId, function (response) {
+                var src = baseUrl + '/generic/widgets?';
+                if (response.widgetid) {
+                    src = src + '&currentWidgetId=' + response.widgetid;
+                }
+                if (response.pageid) {
+                    src = src + '&currentPageId=' + response.pageid;
+                }
+                src = src + '&transparentBackground=1&parentDomain=' + domain;
+                document.getElementById('tawk_widget_customization').src = src;
+
+                jQuery('#excludeurl').val(response.excludeurl);
+                jQuery('#includeurl').val(response.includeurl);
+                if (response.alwaysdisplay == 1) {
+                    jQuery('#alwaysdisplay').prop('checked', true);
+                    jQuery("#exlucded_urls_container").show();
+                } else {
+                    jQuery('#alwaysdisplay').prop('checked', false);
+                    jQuery("#exlucded_urls_container").hide();
+                }
+
+                if (response.donotdisplay == 1) {
+                    jQuery('#donotdisplay').prop('checked', true);
+                    jQuery("#included_urls_container").show();
+                } else {
+                    jQuery('#donotdisplay').prop('checked', false);
+                    jQuery("#included_urls_container").hide();
+                }
+            });
+        }
+     });
 </script>
 
 <label for="websiteids" class="websiteids_label">Select Store</label>
 <select name="websiteids" id="websiteids" class="websiteids">
-<?php echo $this->getWebSiteoptions(); ?>
+<?= /* @noEscape */ $block->getWebSiteoptions(); ?>
```

Comment:
In case anyone wonders why this isn't escaped and has /* @noEscape */.

/* @noEscape */ - this tells the linter that this doesn't need escaping.
$block->getWebSiteoptions() returns the list of <options> tags of the stores existing in the Magento installation. And when I tried to escape it via $block->escapeHtml() it becomes &lt;option value="1"&gt;Store Name&lt;/option&gt; and doesn't render it as HTML code